packages/valory/connections/abci/__init__.py

```diff
@@ -18,3 +18,9 @@
 # ------------------------------------------------------------------------------
 
 """Abci connection."""  # pragma: nocover
+from hypothesis import settings
+
+
+CI = "CI"  # pragma: nocover
+
+settings.register_profile(CI, deadline=5000)  # pragma: nocover
```

Comment:
this is how we have it in `abstract_round_abci`
https://github.com/valory-xyz/open-autonomy/blob/9624a1b64007148192a518239139bdb9c3f5134f/packages/valory/skills/abstract_round_abci/tests/conftest.py#L37-L40